src/Database/Migrator.php

```diff
@@ -271,22 +271,23 @@ public function resolve($path, $file)
     public function installFromSchema(string $path)
     {
         $schemaPath = "$path/install.dump";
+        $prefixedSchemaPath = "$path/install_prefixed.dump.tmp";
```

Comment:
One more question :sweat_smile: 

Isn't this inside of the `vendor` folder? Do we expect that folder to be writable by the user the installer runs as? Do we have access to the storage folder path at that point? Or should we use PHP's `tmpfile` feature?

Reply:
I'm not sure what a better solution would be. Storage folder maybe?

Reply:
Actually we can't do storage, since the paths object might not exist yet.